images/basic/tomcat/Patch.arm64

```diff
@@ -0,0 +1 @@
+FROM arm64v8/tomcat:8.0
```

Comment:
These cases are interesting. Maybe we if we detect a Dockerfile.$ARCH in the directory we will build with that instead? And we can update the ALL_IMAGES to appropriately find only "Dockerfile" (generic) and "Dockerfile.$ARCH" directories. Comment below is related.